client/src/components/Column.js

```diff
@@ -8,55 +8,56 @@ import Grid from "@material-ui/core/Grid";
 import Paper from "@material-ui/core/Paper";
 import Button from "@material-ui/core/Button";
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
   paper: {
     height: 550,
     width: 320,
     background: "#F4F6FF",
     boxShadow: "none",
     position: "relative",
-    marginRight: "20px"
+    marginRight: "20px",
   },
   column: {
     marginLeft: "25px",
-    paddingTop: "20px"
+    paddingTop: "20px",
   },
   columnTitle: {
     display: "flex",
-    alignItems: "center"
+    alignItems: "center",
   },
   icon: {
     marginLeft: "150px",
     marginTop: "20px",
     padding: "20px",
-    cursor: "pointer"
+    cursor: "pointer",
   },
   cardSection: {
     height: 410,
-    overflow: "hidden",
+    position: "relative",
+    overflow: "auto",
     overflowY: "scroll",
     "&::-webkit-scrollbar": {
-      display: "none"
-    }
+      display: "none",
+    },
   },
```

Comment:
Try this:
```js
 cardSection: {
    height: 410,
    zIndex: 0,
    "&:before": {
      zIndex: 1,
      background:
        "linear-gradient(180deg, rgb(245, 246, 254) 0%, rgba(9,9,121,0) 31%)",
      top: "13%",
      content: '""',
      display: "block",
      height: "60px",
      left: "0",
      pointerEvents: "none",
      position: "absolute",
      width: "100%",
    },
    "&:after": {
      zIndex: 1,
      background:
        "linear-gradient(0deg, rgb(245, 246, 254) 0%, rgba(9,9,121,0) 31%)",
      bottom: "11%",
      content: '""',
      display: "block",
      height: "60px",
      left: "0",
      pointerEvents: "none",
      position: "absolute",
      width: "100%",
    },
    overflow: "auto",
    overflowY: "scroll",
    "&::-webkit-scrollbar": {
      display: "none",
    },
  },
```

Reply:
thank you for this @aeciorc , this worked. The changes have been made